code/modules/mob/living/carbon/carbon_update_icons.dm

```diff
@@ -302,6 +358,29 @@
 
 	return .
 
+/**
+ * Generates a cache key for masks (mainly only used for right legs now, but perhaps in the future...).
+ *
+ * This is exactly like generate_icon_key(), except that it doesn't add `"-[draw_color]"`
+ * to the returned list under any circumstance. Why? Because it (generate_icon_key()) is
+ * a proc that gets called a ton and I don't want this to affect its performance.
+ *
+ * Returns a list of strings.
+ */
+/obj/item/bodypart/proc/generate_mask_key()
+	RETURN_TYPE(/list)
+	. = list()
```

Comment:
rather then using . as a var, you should just make a new list called like, to_return or something.
If you're just going to treat . as a var all the time you'd better off not using . (unless your code is hideously hot, which this isn't)

Reply:
I'll be honest, I just copy-pasted this from Kapu's code, I don't mind changing it, I just kept it this way for consistency.

Reply:
Kapu seems to be plannin on changing them. I'd like it if you could change it here tho